src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetCaseViewOperation.java

```diff
@@ -110,4 +118,21 @@ private CaseView filterUpsertAccess(String caseReference,
 
         return caseView;
     }
+
+    @Override
+    public CaseView updateWithAccessControlMetadata(CaseView caseView) {
+        List<CaseViewField> metadataFields = caseView.getMetadataFields();
+
+        CaseAccessMetadata caseAccessMetadata = caseDataAccessControl.generateAccessMetadata(caseView.getCaseId());
+
+        CaseViewField caseViewField = new CaseViewField();
+        caseViewField.setId(ACCESS_GRANTED);
+        caseViewField.setLabel(ACCESS_GRANTED_LABEL);
+        caseViewField.setSecurityLabel(SecurityClassification.PUBLIC.name());
+        caseViewField.setValue(caseAccessMetadata.getAccessGrants());
+
+        metadataFields.add(caseViewField);
+        caseView.setMetadataFields(metadataFields);
```

Comment:
Calling `addMetadataField(..)` is better ? 
It looks like it doesn't exist. Do you mind to add that method please?